lib/node_modules/@stdlib/math/base/special/gammasgn/docs/types/test.ts

```diff
@@ -0,0 +1,44 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import gammasgn = require( './index' );
+
+
+// TESTS //
+
+// The function returns a number...
+{
+	gammasgn( 2 ); // $ExpectType number
+}
+
+// The function does not compile if provided a value other than a number...
```

Comment:
```suggestion
// The compiler throws an error if the function is provided a value other than a number...
```

At some point, we should standardize these comments across the project.